### PR TITLE
feature: ZENKO-367 CRR to AWS end-to-end tests

### PR DESCRIPTION
**Pull request by bennettbuchanan**
This PR adds CRR to AWS replication end-to-end tests. For now, I've skipped tests which were previously passing but are not currently in Zenko—these skipped tests require further investigation.

### CODE REVIEW

**Comment by bennettbuchanan** [comment]
@bert-e wait

**Comment by ssalaues** [comment]
Can you rebase off master? We merged in some fixes to the CI this morning

**Comment by bennettbuchanan** [comment]
@ssalaues I've rebased to master, thanks!

File: tests/zenko_e2e/backbeat/s3SDK.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const S3 = require('aws-sdk').S3;
+
+const scalityS3Client = new S3({
+    accessKeyId: process.env.ZENKO_BACKBEAT_ACCESS_KEY,
+    secretAccessKey: process.env.ZENKO_BACKBEAT_SECRET_KEY,
+    sslEnabled: false,
+    endpoint: 'http://127.0.0.1:8000',
```
Comment (ssalaues) [inline tests/zenko_e2e/backbeat/s3SDK.js]: Do we assume that, when the tests are ran, it's expected to have 8000 port-forwarded to localhost? Why not use `zenko.local` ?

Comment (ssalaues) [reply tests/zenko_e2e/backbeat/s3SDK.js]: Either way is okay really but might be good to have a note of it in the doc.

**Pull request merged**